iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/variant_internal.hpp

```diff
@@ -91,8 +91,10 @@ struct get_type_at_index<N, N, T, Targs...>
     using type = T;
 };
 
-// AXIVION DISABLE STYLE AutosarC++19_03-M5.2.8, AutosarC++19_03-A5.2.4 : conversion to typed pointer is intentional, it is correctly aligned and points to sufficient memory for a T by design of variant; note that this is an internal class used by variant
-// AXIVION DISABLE STYLE AutosarC++19_03-A18.5.10, FaultDetection-IndirectAssignmentOverflow : destination is aligned to the maximum alignment of Types, see variant.hpp
+// AXIVION DISABLE STYLE AutosarC++19_03-M5.2.8 : conversion to typed pointer is intentional, it is correctly aligned and points to sufficient memory for a T by design of variant; note that this is an internal class used by variant
+// AXIVION DISABLE STYLE AutosarC++19_03-A5.2.4 : conversion to typed pointer is intentional, it is correctly aligned and points to sufficient memory for a T by design of variant; note that this is an internal class used by variant
```

Comment:
Could potentially also just be

```suggestion
// AXIVION DISABLE STYLE AutosarC++19_03-A5.2.4 : see above
```

No hard feelings, though.